common/plugins/org.polarsys.capella.common/src/org/polarsys/capella/common/helpers/EObjectLabelProviderHelper.java

```diff
@@ -106,6 +107,9 @@ public static String getMetaclassLabel(EObject object, boolean addParenthesis) {
     if (null == object) {
       return label;
     }
+    if(object instanceof DAnnotation && ((DAnnotation)object).getSource().contains("TitleBlock")) {
```

Comment:
If I use the helper (TitleBlockHelper), I introduce a cyclic dependency. I return empty String because otherwise in Properties view it displays "(DAnnotaion) Diagram Title Block" in the title.

Reply:
I undo changes in this file, moved it to Title Block providers but will show the meta class Title Block.